docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,51 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses several CI build servers such as Centos, AppVeyor, Circle and Travis. These servers build incoming pull requests, pass it through various test parameters and provide the status of the builds to ensure continuous integration and deployment. Except for Travis CI, they also archive build artifacts of successful builds.
```

Comment:
Why "such as"? IMO such as is used to provide a list of examples. But here we give the full known list of CI servers we use. This are not just examples.